### DISCUSSION TITLE
How to de-activate Stop & Start by default

### DISCUSSION THREAD

**Discussion by RiskyBussiness** [cedricp/ddt4all#966] [Q&A]
Hello, when I start my car, S&S function is by default activated. I don't want to inhibit S&S altogether, but I don't want it to be activated as soon as I start my car. So, does anybody know how to de-activate it when the car starts? This way, I can turn it on whenever I want to use it.

**Comment by KarelSvo** [comment]
[Click here](https://www-dacianer-de.translate.goog/threads/start-stopp-automatisch-ausschalten-so-gehts.25348/?_x_tr_sl=auto&_x_tr_tl=en&_x_tr_hl=en&_x_tr_pto=wapp)

**Reply by DavidIgor** [reply]
I just shorted bonnet switch connector(near bonnet locking mechanism) and Start stop is disabled. car thinks that bonnet is open and start stop is off ....  I was thinking of running couple of wires inside and installing simple switch....

**Reply by AndyIreland** [reply]
I think there is an extra wire on the battery that can control S&S too - a thinner cable. Some people remove that to disable S&S

**Reply by matzel82** [reply]
You only disconnect the plug at the sensor at the battery - pole

**Reply by DavidIgor** [reply]
you mean that sensor at negative battery pole? I was in it is safety pyro fuse .

**Comment by KarelSvo** [comment]
> I think there is an extra wire on the battery that can control S&S too - a thinner cable. Some people remove that to disable S&S

This is only done by users who do not know how the alternator charge control works.  S&S OFF is just a byproduct of faulty charge control.

**Reply by DavidIgor** [reply]
I'll stick with my bonnet switch mod. @KarelSvo any problems with it for long run ? car will be happy right? (I driving like that for 3-4 months now)

**Comment by ShaneHash** [comment]
Old thread, I know...

I unplugged and removed the switch that checks if the bonnet is open, then plugged it back in again, but with the switch tied away from everything with a cable tie. As far as the car ECU is concerned, the bonnet is open even when it really isn't. As a result, stop start never kicks in.
